library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -121,7 +123,9 @@ private void deleteFrom(SqlDatabaseWrapper database, Migration migration) {
         for (Migration.FileMetadata metadata : migration.getFileMetadata()) {
             if (hasValidFileLocation(metadata)) {
                 File file = new File(metadata.originalFileLocation());
-                file.delete();
+                boolean deleted = file.delete();
```

Comment:
ATM, as far as I know, we hope that we are deleting files correctly. Perhaps we should raise an issue to deal with failures?

Reply:
yeah I know that one, its usually ignored as not much can be done though

Reply:
Yea that's why I found the warning kind of annoying. Short of surfacing to a user there isn't much we can do 😂